gmso/core/virtual_type.py

```diff
@@ -0,0 +1,400 @@
+import warnings
+from typing import Callable, Optional, Set, Tuple, Union
+
+import unyt as u
+from pydantic import ConfigDict, Field, field_serializer, field_validator
+
+from gmso.abc.gmso_base import GMSOBase
+from gmso.abc.serialization_utils import unyt_to_dict
+from gmso.core.parametric_potential import ParametricPotential
+from gmso.utils._constants import UNIT_WARNING_STRING
+from gmso.utils.expression import PotentialExpression
+from gmso.utils.misc import ensure_valid_dimensions, unyt_compare
+from gmso.utils.units import GMSO_UnitRegistry
+
+
+class VirtualPositionType(ParametricPotential):
+    """A description of the interaction between 3 bonded partners.
+
+    This is a subclass of the gmso.core.Potential superclass.
+
+    AngleType represents an angle type and includes the functional form
```

Comment:
Looks like we still have some doc strings left over from copying the AngleType class.

Reply:
Fixed in 25ad196